Equality.Core/Models/Interfaces/IColumn.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Equality.Models
+{
+    public interface IColumn
```

Comment:
Structure is not full (see https://github.com/indiebox/equality-api/blob/master/database/migrations/2022_03_13_085034_create_columns_table.php and https://github.com/indiebox/equality-api/blob/master/database/migrations/2022_03_19_101149_add_order_column_to_columns_table.php)